src/applications/disability-benefits/all-claims/tests/utils/addToxicExposure.unit.spec.jsx

```diff
@@ -0,0 +1,305 @@
+import { expect } from 'chai';
+import cloneDeep from 'platform/utilities/data/cloneDeep';
+import { addToxicExposure } from '../../utils/submit';
+
+describe('addToxicExposure', () => {
+  describe('when user has no toxic exposure data', () => {
+    it('should return the form data unchanged', () => {
+      const formData = {
+        newDisabilities: [{ condition: 'Chronic Bronchitis' }],
+        veteranInformation: { name: 'John Doe' },
+      };
+
+      const result = addToxicExposure(formData);
+
+      expect(result).to.deep.equal(formData);
+    });
+  });
+
+  describe('when user has toxic exposure data but no conditions', () => {
```

Comment:
Perhaps modify the description to specify the user has no "toxic exposure" conditions.